tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/model/JmapSettingsGet.scala

```diff
@@ -11,25 +11,25 @@ case class JmapSettingsGet(accountId: AccountId,
                            ids: Option[Set[JmapSettingsId]] = None) extends WithAccountId
 case class JmapSettingsResponse(accountId: AccountId,
                                 state: UuidState,
-                                list: Seq[JmapSettingsObject],
+                                list: Seq[JmapSettingsEntry],
                                 notFound: Seq[JmapSettingsId])
 
-object JmapSettingsObject {
+object JmapSettingsEntry {
```

Comment:
`JmapSettingsEntry` -> `JmapSettingsObject` in the `ISSUE-788 Setting to enable Push for firebase (#794)` commit. To better distinguish with `trait JmapSettingEntry `.

Why change this back?

Reply:
Bacause I see the latest commit use `JmapSettingsEntry`. I will change it into `JmapSettingsObject`.